### ISSUE TITLE
{Linked:Bug1654480} [Adaptive Cards > Designer] - Screen Reader is not announcing the descriptive information for the 'Register' link. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8794]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/Action.ToggleVisibility.html
OS Version: 23H3(OS Build 26020.1000)
Browser Version: Edge Dev 122.0.2325.0
Screen Reader: NVDA/Narrator

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Run the Tool and observe the issue.

**Actual result:**
Aria - Label for the 'Register' link is not descriptive due to this screen reader is announcing as **'Register Link.'**

**Expected Result:**
Aria-Label should be more descriptive for the 'Register' link and while navigating to it, screen reader should announce as **'Register for Microsoft Azure Administrator Link'.**

**User Impact:**
It will impact the screen reader user as they will not able to know the actual purpose of the the register button if the Register link will not descriptive.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

![Register link should be more descriptive](https://github.com/microsoft/AdaptiveCards/assets/95913748/a354d5a7-54f1-46ec-93c9-a8f1cf5c339a)

### Card JSON

```JSON
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "Container",
            "items": [
                {
                    "type": "TextBlock",
                    "size": "large",
                    "text": "Our recommendation for you!",
                    "color": "accent",
                    "wrap": true
                },
                {
                    "type": "TextBlock",
                    "text": "Based on your organizational needs",
                    "wrap": true
                }
            ]
        },
        {
            "type": "Container",
            "items": [
                {
                    "type": "TextBlock",
                    "text": "[Microsoft Azure Administrator]()",
                    "weight": "Bolder",
                    "wrap": true
                },
                {
                    "type": "RichTextBlock",
                    "inlines": [
                        {
                            "type": "TextRun",
                            "text": "Date: "
                        },
                        {
                            "type": "TextRun",
                            "text": "Dec 04, 2023 - Dec 29, 2023",
                            "weight": "Bolder"
                        }
                    ]
                },
                {
                    "type": "RichTextBlock",
                    "inlines": [
                        {
                            "type": "TextRun",
                            "text": "Time: "
                        },
                        {
                            "type": "TextRun",
                            "text": "09:00 AM - 05:00 PM (GMT+01:00) Brussels, Copenhagen, Madrid, Paris",
                            "weight": "Bolder"
                        }
                    ]
                },
                {
                    "type": "RichTextBlock",
                    "inlines": [
                        {
                            "type": "TextRun",
                            "text": "Language: "
                        },
                        {
                            "type": "TextRun",
                            "text": "English",
                            "weight": "Bolder"
                        }
                    ]
                },
                {
                    "type": "RichTextBlock",
                    "inlines": [
                        {
                            "type": "TextRun",
                            "text": "Delivery ID: "
                        },
                        {
                            "type": "TextRun",
                            "text": "36877",
                            "weight": "Bolder"
                        }
                    ]
                },
                {
                    "type": "ActionSet",
                    "actions": [
                        {
                            "type": "Action.OpenUrl",
                            "title": "Register ",
                            "label": "Register link opens in a new tab",
                            "url": "#"
                        }
                    ]
                }
            ]
        }
    ],
    "actions": [
        {
            "type": "Action.OpenUrl",
            "title": "View More \uF35F",
            "url": "#"
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.3"
}
```

**Comment by vagpt** [comment]
This issue is originally tracked with below URL.

https://dev.azure.com/worldwidelearning/WWL%20Accessibility/_workitems/edit/1654480

**Comment by vagpt** [comment]
Hi @anna-dingler,

Any update on this bug?

**Comment by anna-dingler** [comment]
Hi @vagpt,

I spoke with my manager, and we do not have the capacity to work on Sev3 bugs at this time.

-Anna

**Comment by jwoo-msft** [comment]
[like]  Joseph Woo reacted to your message:
________________________________

**Comment by vagpt** [comment]
As per the comment above, closing this issue.
